src/cmd/services/m3coordinator/downsample/downsampler.go

```diff
@@ -80,11 +80,13 @@ type SamplesAppenderOverrideRules struct {
 // SamplesAppender is a downsampling samples appender,
 // that can only be called by a single caller at a time.
 type SamplesAppender interface {
-	AppendCounterSample(value int64) error
-	AppendGaugeSample(value float64) error
-	AppendCounterTimedSample(t time.Time, value int64) error
-	AppendGaugeTimedSample(t time.Time, value float64) error
-	AppendTimerTimedSample(t time.Time, value float64) error
+	AppendUntimedCounterSample(value int64) error
```

Comment:
These are only used by tests. I am going to check if they are even needed and remove them - maybe in a subsequent change. Will add a note reflecting the same.